### PR TITLE
Gfeitosa/enable int improve coverage

### PR DESCRIPTION
**Pull request by gfeitosa-msft**
[Task 3425822](https://skype.visualstudio.com/SPOOL/_workitems/edit/3425822): [Reliability] Improve test coverage for Python SDK

Added int environment to Python livetest pipelines:

image

Enabled branch coverage reporting for all SDKs, and both line and branch coverage are above 80%. Also removed shared resources from coverage report so we check only coverage of our own code.

image

Yaml pipeline definition updated to use SMS specific configuration for both int and prod environments, so we don't use shared resources or phone numbers, reducing risk of impact from other teams' actions. 

Connection string env var updated to be consistent with the other SDKs.

### CODE REVIEW

**Review by phermanov-msft — APPROVED** [review]

**Review by ilyapaliakou-msft — APPROVED** [review]

File: eng/tox/tox.ini [hunk 1]
```diff
@@ -48,7 +48,7 @@ pkgs =
 
 [pytest]
 ignore_args=--ignore=.tox --ignore=build --ignore=.eggs
-default_args = -rsfE --junitxml={tox_root}/test-junit-{envname}.xml --verbose --durations=10 --ignore=azure {[pytest]ignore_args}
+default_args = -rsfE --junitxml={tox_root}/test-junit-{envname}.xml --verbose --cov-branch --durations=10 --ignore=azure {[pytest]ignore_args}
```
Comment (benbp) [inline eng/tox/tox.ini:51]: FYI @scbedd

**Review by scbedd — APPROVED** [review]
Thank you for the update in `tox.ini`!

**Pull request merged**